<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceIdApplication</class>
 <widget class="QMainWindow" name="VoiceIdApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>714</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="insertRB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Insert your voice</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="checkRB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Check your voice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 175, 255);</string>
    </property>
    <property name="text">
     <string>Record voice</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter your name</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="deleteRB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Delete voice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Delete voice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Check voice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Insert voice</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultWindow">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
